libyul/backends/evm/StackLayoutGenerator.cpp

```diff
@@ -767,10 +767,61 @@ void StackLayoutGenerator::fillInJunk(CFG::BasicBlock const& _block)
 		createStackLayout(_source, _target, swap, dupOrPush, pop);
 		return opGas;
 	};
+
+	if (_functionInfo && !_functionInfo->canContinue && _block.allowsJunk())
+	{
+		// TODO: unify with the one below
```

Comment:
If anyone wants to finish this while I'm off: this is pretty much the only thing left to do here.

Reply:
so I guess not so much and could be done soon? :D